backend/src/BuiltinExecution/Libs/List.fs

```diff
@@ -493,8 +490,8 @@ let fns : List<BuiltInFn> =
       previewable = Pure
       deprecated = NotDeprecated }
 
-
-    { name = fn "filter" 0
+    // to delete?
```

Comment:
delete comments + add a question in the pr (I think we have equivalent functions in dark)